cicd/gitops/getting-started-with-openshift-gitops.adoc

```diff
@@ -0,0 +1,19 @@
+[id="getting-started-with-openshift-gitops"]
+= Getting started with OpenShift GitOps
+include::modules/common-attributes.adoc[]
+//include::modules/gitops-document-attributes.adoc[]
```

Comment:
I added the `//` before this include because this doc does not exist in this PR. This is the correct way to prevent build errors from occurring, right?

Reply:
We would want to use {gitops-title} in this assembly too, so I am not sure I understand why we want to comment this out.